package.xml

```diff
@@ -26,28 +26,21 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>eigen</build_depend>
-  <build_depend>kdl_parser</build_depend>
+  <depend>kdl_parser</depend>
+  <depend>rosconsole</depend>
+  <depend>roscpp</depend>
+  <depend>rostime</depend>
+  <depend>sensor_msgs</depend>
+  <depend>tf</depend>
+  <depend>tf2_ros</depend>
+  <depend>tf2_kdl</depend>
+
   <build_depend>liborocos-kdl-dev</build_depend>
-  <build_depend>rosconsole</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>rostime</build_depend>
-  <build_depend>sensor_msgs</build_depend>
-  <build_depend>tf2_ros</build_depend>
-  <build_depend>tf2_kdl</build_depend>
   <build_depend>liburdfdom-headers-dev</build_depend>
-  <run_depend>catkin</run_depend>
-  <run_depend>eigen</run_depend>
-  <run_depend>kdl_parser</run_depend>
-  <run_depend>liborocos-kdl-dev</run_depend>
-  <run_depend>rosconsole</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>rostime</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>tf2_ros</run_depend>
-  <run_depend>tf2_kdl</run_depend>
+  <build_depend>eigen</build_depend>
 
-  <test_depend>rosbag</test_depend>
+  <exec_depend>liborocos-kdl-dev</exec_depend>
```

Comment:
Shouldn't this be:

`<exec_depend>liborocos-kdl</exec_depend>`

Reply:
`<build_export_depend>liborocos-kdl-dev` because headers are included in headers we install, and `<exec_depend>liborocos-kdl` in da6f692513c924a939946fa13dce3b017dc6a41f